### DISCUSSION TITLE
NFTs Renting System

### DISCUSSION THREAD

**Discussion by poteitouweb3** [sunflower-land/sunflower-land#2044] [Ideas]
### **Introduction**

I had already suggested it in discord but I would like to formulate it better and start this discussion here as well.

As we know, what makes certain NFTs unique is not only their scarcity but mainly their benefits (buffs) generated for the player in-game. However for some NFTs with enviable buffs the price to permanently possess one of these NFTs is also high and unattainable for most players. Based on this I would like to suggest a way to loan NFTs via contract.

### **How it would work**

The most viable option would be not to transfer the NFT itself but from the smart contract assign to the farm the buff for the given time. E.g.

- Player A wants to rent his foreman beaver for 3 SFL/hr;
- There would be a interface similar to the marketplace listing the rentals and pertinent information;
- Player B would choose how many hours he would like to rent the NFT and then proceed to sign the smart contract;
- Upon signing, player A would receive the $SFL and temporarily lose his foreman beaver buff and player B would hand over the $SFL and receive the buff and a "purely visual" foreman beaver for the agreed time;

### **Conclusion**

I believe that the system would not depreciate the prices/ownership value of NFTs. In fact quite the contrary as now the owner will be able to passively generate $SFL with his NFTs.

### **Points to be discussed**

- Should there be minimum and maximum times and values for rentals?

- How much will the fee be charged by the game? A percentage of the rent or a fixed amount as in the market?

- If the player has more than one NFT can he keep the buff of one and rent the others?

- Will it be possible to generate a "private contract" out of the public range for renting, like e.g. in cases where a friend wants to rent/loan the nft to another in a safe way?

**Comment by TheDcol** [comment]
Nice points and good discussion to bring on, saw a few people talk about it more recently.

I am not against renting nfts, but I am also not for it.. personally.

I think in a game if you want an item it is because you want it for yourself, not because you can rent it out to earn SFL passively, you yourself should earn the SFL by using it.

It would also cause scenarios where a few people could buy up the majority of an nft supply and set the prices themselves for rentals, this can technically happen now (and it does happen) with people buying multiples of nfts to sell for profit later, but I think it would happen a lot more with renting.

Is the goal to allow new players to grow their farm quicker by spending x sfl to rent these nfts or for the playey to earn sfl by renting out nfts they do not use?

If it's the former, I think introduction of cheap nfts for new players would be better. I see nft renting akin to scholarships in a way and I am very against that, but curious to see what others think :)

Well explained mate, regardless of what I think :D
